### PR TITLE
simplified headings

### PR DESCRIPTION
**Pull request by stschiff**
I wanted to look up the documentation for a specific sub-command in trident, and was annoyed to not see any structure in the sidebar of our webpage when I clicked on "trident". I think this was a relic from when we had the tabs for the different versions. Now, with the neat PDF link list, we can switch them back on. This PR does that.

In doing so, I also took the liberty to reduce some of the heading depth, by removing unnecessary solitary headlines (headlines that are the only headline at a specific level)

### CODE REVIEW

**Comment by nevrome** [comment]
The headline depth is also tuned for the .pdf output. I'm a bit worried that shifting them may have detrimental effects. Did you check how the .pdf output looks with these changes?

**Comment by nevrome** [comment]
I'm all good with changing things as long as the output for all rendering targets is still readable.

**Comment by stschiff** [comment]
Right, how do I check the PDFs? They are created with a GitHub action only when I push to Master, right? Do you have an idea how to test this?

**Comment by nevrome** [comment]
I think you can just follow the steps performed in the GitHub Action (as documented [here](https://github.com/poseidon-framework/poseidon-framework.github.io/tree/master/pdf_conversion)). For you it probably boils down to 

1. Run a hacky sed commands on the relevant files you want to render: https://github.com/poseidon-framework/poseidon-framework.github.io/blob/e6420d88771cc36fb3115a4c313707adc8c9986a/.github/workflows/addPDF.yml#L30-L36
or quickly edit the files manually.

2. Run pandoc (v3.1.12.2-1) on them with [this](https://github.com/poseidon-framework/poseidon-framework.github.io/blob/master/pdf_conversion/pandoc_pdf_config.yml) config file, which also requires some other files in this directory.

**Comment by stschiff** [comment]
Alright, so I'm now at a stage where things look good, but I need feedback.

The goal of this PR was to set up the documentation sections for the command line tools in such a way that the sidebar on the website shows useful sub-headings. This meant that I had to simplify and flatten the heading-hierarchy a bit. This worked and in my view looks nice and is useful (for example, the many different trident subcommands are now nicely listed in the sidebar for quick navigation).

In order to preserve the nice automatic PDF generation (chapeau for making this work, @nevrome), I had to tweak the GitHub action a bit, as you can see.

I also introduced fake headings (like `<h1>trident CLI software</h1>`). This is a hack: The sidebar heading-structure ignores the first heading starting with `#`, which in our case is then `# Guide for trident v1.5.4.0`. So it's a trick how both of these top-level headings get ignored. We could also remove the top-heading `<h1>trident CLI software</h1>` altogether, but it looks nicer to have it.

What I need feedback on, is the fact that I did _not_ so far fix the old archive-source files (in `trident_guide_archive` and the other two). They render alright in the PDFs, but the heading structure has not been adapted, so they now end up without a central heading on top, and many sub-headings. I find this acceptable. But if you think it's necessary, I can go through all of the source file and adapt the heading depths. This is not so much work, and I'm in principle happy to do it, but I first wanted to see whether you think it's necessary.

Let me know.

File: xerxes.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <popup :custom-text="`<p><a href='https://mpi-eva-archaeogenetics.github.io/comp_human_adna_book/fstats.html'>Introduction to F3- and F4-Statistics</a> by Stephan Schiffels: An explanation of F-Statistics and how to run them with xerxes</p>`"></popup>
```
Comment (nevrome) [inline xerxes.md:1]: xerxes.pdf was not affected by these changes?

**Review by nevrome — APPROVED** [review]
Checked everything quickly. The website headings are there and useful. The .pdfs look fine - also the old ones, so imho you don't have to go through them again.

File: pdf_tests/trident.md [hunk 2]
```diff
@@ -0,0 +1,1040 @@
+# Guide for trident v1.5.4.0
```
[the remark this answers is not in the document]
Comment (stschiff) [reply pdf_tests/trident.md:1]: Oops, yes of course. Will fix

File: xerxes.md (hunk 1 above)
Comment (stschiff) [reply xerxes.md:1]: Hmmm, what is this? I will check.

Comment (stschiff) [reply xerxes.md:1]: I'm actually not following what you're getting at? That line (Nr. 1) did not change, right? This whole part above the first heading is removed by the sed-command for the PDF.

Comment (nevrome) [reply xerxes.md:1]: Sorry - that was not clear. I saw that xerxes.pdf (and actually also janno_details.pdf) did not change, but trident.pdf did. I was just wondering why some of the changes in the .md files had no consequences in the .pdf files but others.

Comment (nevrome) [reply xerxes.md:1]: But the [GitHub Action](https://github.com/poseidon-framework/poseidon-framework.github.io/actions/runs/10490127511/job/29056190508) looks fine. So I guess they did get rendered properly and just happend to be identical to the previous version.

Comment (stschiff) [reply xerxes.md:1]: Right. OK.

**Pull request merged**